eth/examples/Compound/borrow.ts

```diff
@@ -10,8 +10,8 @@ const borrowTokenName = 'USDC';
 const compoundTokenName = 'cUSDC';
 const borrowAmount = 1e5;
 
-const compoundComptroller = new Contract('CompoundComptroller'); // there is only 1, so no need for instance
-const compoundCollateralTokenContract = new Contract('Compound').instance(collateralTokenName);
+const compoundComptroller = getContractsFactory('eth').getContract('CompoundComptroller'); // there is only 1, so no need for instance
```

Comment:
similarly, maybe just `.contract` instead of `.getContract`

Reply:
Same here. `contract('CompoundComptroller')`reads as if you are setting the contract